packages/app-store/salesforce/zod.ts

```diff
@@ -3,6 +3,13 @@ import { z } from "zod";
 import { eventTypeAppCardZod } from "../eventTypeAppCardZod";
 import { SalesforceRecordEnum } from "./lib/recordEnum";
 
+export const routingFormOptions = z
```

Comment:
I think routing form app data should be separate from the other app data. What do you think?

Reply:
Yeah makes sense